tf-modules/cumulus/ingest.tf

```diff
@@ -8,6 +8,14 @@ module "ingest" {
   distribution_url                         = module.distribution.distribution_url
   cumulus_message_adapter_lambda_layer_arn = var.cumulus_message_adapter_lambda_layer_arn
 
+  # API config
+
+  archive_api_uri       = module.archive.internal_api_uri
```

Comment:
I... suppose so.   To my mind, as far as the ingest (not part of the monolith) tf module is concerned there is no "internal/external" API URL.

In fact, I'm thinking that it's the change to archive locals/outputs that's the problem.   How about setting that to have an api_uri and an external_api_uri instead?

Reply:
I'm not sure I follow. The ingest module is included as part of the Cumulus module monolith? 

I think it's confusing because the input variable to `ingest` is named `archive_api_uri`, but it is **not** supposed to accept the `api_uri` output from the `archive` module. 

We can discuss offline

Reply:
Sure, let me know when you're available

Reply:
Talked offline, will update to internal_archive_api_uri

Reply:
36ca4f94e